GPflow/ekernels.py

```diff
@@ -0,0 +1,307 @@
+from functools import reduce
+import warnings
+import tensorflow as tf
+from . import kernels
+from .tf_wraps import eye
+from ._settings import settings
+
+int_type = settings.dtypes.int_type
+float_type = settings.dtypes.float_type
+
+# TODO: Efficiency gain from using diagonal q(X).
```

Comment:
Sure, do you mind keeping the TODO's in the code as well? Or would you rather have them removed?